### ISSUE TITLE
Update the CSS to make the header fully universal

### ISSUE THREAD

**Issue by EdwinGuzman** [NYPL/dgx-header-component#38]
NYPLBase CSS rules, Drupal CSS rules, Design Toolkit CSS rules, and other CSS rules all change the header in very minute ways in their own respective environments. We can try to make the header more isolated in its CSS but we also need to consolidate and have one CSS base and styleguide to follow.

CC @ricardoom @rhernand3z @ktp242

**Comment by ricardoom** [comment]
Agreed. Getting there will be difficult w/ many obstacles, the code being the least of it...
